packages/web-awesome/src/components/TestResult/TestResultSetup/index.tsx

```diff
@@ -17,18 +18,26 @@ const typeMap = {
 
 export type TestResultSetupProps = {
   setup: AllureAwesomeTestResult["setup"];
+  id?: string;
 };
 
-export const TestResultSetup: FunctionalComponent<TestResultSetupProps> = ({ setup }) => {
-  const [isOpened, setIsOpen] = useState(false);
+export const TestResultSetup: FunctionalComponent<TestResultSetupProps> = ({ setup, id }) => {
+  const teardownId = `${id}-setup`;
+  const isEarlyCollapsed = Boolean(!collapsedTrees.value.has(teardownId));
```

Comment:
Method `has` already returns `boolean` value, there is no need to have additional conversion here